docs/current_docs/quickstart/snippets/simplify/php/src/HelloDagger.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DaggerModule;
+
+use Dagger\Attribute\DaggerFunction;
+use Dagger\Attribute\DaggerObject;
+use Dagger\Attribute\Doc;
+use Dagger\Container;
+use Dagger\Directory;
+
+use function Dagger\dag;
+
+#[DaggerObject]
+class HelloDagger
+{
+    #[DaggerFunction]
+    #[Doc('Publish the application container after building and testing it on-the-fly')]
+    public function publish(Directory $source): string
+    {
+        $this->test($source);
+
+        return $this
+            ->build($source)
+            ->publish('ttl.sh/hello-dagger-'.rand(0, 10000000));
+    }
+
+    #[DaggerFunction]
+    #[Doc('Build the application container')]
+    public function build(Directory $source): Container
+    {
+        $build = dag()
+            ->node(null, $this->buildEnv($source))
+            ->commands()
+            ->run(['build'])
+            ->directory('./dist');
+
+        return dag()
+          ->container()
+          ->from('nginx:1.25-alpine')
+          ->withDirectory('/usr/share/nginx/html', $build)
+          ->withExposedPort(80);
+    }
+
+    #[DaggerFunction]
+    #[Doc('Return the result of running unit tests')]
+    public function test(Directory $source): string
+    {
+        return dag()
+            ->node(null, $this->buildEnv($source))
+            ->commands()
+            ->run(['test:unit', 'run'])
```

Comment:
this is incorrect, what is `test:unit run` - makes no sense

```
->withExec(['vendor/bin/phpunit'])
```

is correct

Reply:
@dragoonis the quickstart walkthrough is using [this repo](https://github.com/dagger/hello-dagger) which is a node project, hence the npm specific test call.